README.md

```diff
@@ -1,2 +1,34 @@
-# HelloMicroverse
-This is a hello microverse week one project
+# Hello Microverse
+
+> This is the first project at Microverse first week activity.
+
+## Built With
+
+- HTML 
+- CSS
+
+## Authors
+
+👤 **Aster Alemu**
+
+- GitHub: [aster alemu](https://github.com/aster-alemu)
+- Twitter: [Aster Alemu](https://twitter.com/asteralemu)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/aster-alemu)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Great thanks to one of the Micronat Tarikwa Tesfa she helps me a lot
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] `MIT` link is broken, please consider adding `MIT` License file in the project in order to fix the link.

Reply:
As requested by the previous code reviewer, MIT link is broken, please consider adding the MIT License file in the project in order to fix the link.